### PR TITLE
fix(permissions): alpha role has all full features

### PR DESCRIPTION
**Pull request by dpgaspar**
### SUMMARY
Currently Alpha role does not include the "Manage" menu and it's sub items. But user's still have underlying access to the ModelView's that are included on the "Manager" menu. This PR proposes adding "Manage" menu and their sub items to the "Alpha" Role, making it an "All features included" role, except for security and SQLLab.

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [X] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by mistercrunch** [comment]
By adding the MVs in `ADMIN_ONLY_VIEW_MENUS` you're removing access to `Alpha` and `Gamma`. I don't think that's what we want

**Comment by dpgaspar** [comment]
That was my objective, since by removing the "Manage" menu item, it's kind of implied the user does not have access to the `AnnotationsLayerModelView, `CssTemplateModelView`, `QueryView` but he does.

But seems like we are using the old API for the annotations. Would be awesome to instead of removing we would add the "Manager" menu item to Alpha, that would make it more consistent, and inline with the idea that an Alpha user has access to all features except security

**Comment by mistercrunch** [comment]
I think users still access the API in some other ways though (dashboard / explore / SQL Lab), we'd have to check/confirm whether they hit the modelview endpoints or not.

**Comment by dpgaspar** [comment]
Max, I've reverted the direction here. I'm adding the Manage menu and it's subitems to the alpha role. What do you think?

File: superset/security/manager.py [hunk 1]
```diff
@@ -139,7 +138,13 @@ class SupersetSecurityManager(  # pylint: disable=too-many-public-methods
         "RowLevelSecurityFiltersModelView",
     } | USER_MODEL_VIEWS
 
-    ALPHA_ONLY_VIEW_MENUS = {"Upload a CSV"}
+    ALPHA_ONLY_VIEW_MENUS = {
```
Comment (bkyryliuk) [inline superset/security/manager.py:141]: we do the same in the permission sync script.
1 suggestion, maybe add a unit test 
example:
```
def test_sync_creator_role(setup):
    # type: (Any) -> None
    session = db.create_scoped_session()
    sync_creator_role(session)
    creator_role = session.query(ab_models.Role).filter_by(name="Creator").one()
    assert set([p.view_menu.name for p in creator_role.permissions]) == {
        "TableModelView",
        "TableColumnInlineView",
        "SqlMetricInlineView",
        # TODO(bogdankyryliuk): share the same config with production.
        # "DashboardEmailScheduleView",
        # "SliceEmailScheduleView",
        "Manage",
        "AnnotationModelView",
        "AnnotationLayerModelView",
        "all_query_access",
        "CsvToDatabaseView",
        "Datasource",
        "Upload a CSV",
    }

    assert set([p.permission.name for p in creator_role.permissions]) == {
        "all_query_access",
        "can_add",
        "can_save",
        "can_delete",
        "can_edit",
        "can_external_metadata",
        "can_get",
        "can_list",
        "can_mulexport",
        "can_show",
        "can_this_form_get",
        "can_this_form_post",
        "muldelete",
        "refresh",
        "yaml_export",
        "menu_access",
    }```

**Comment by mistercrunch** [comment]
+1 on a unit test, do we have a doc somewhere that give an idea of what `Gamma` is? Also should have a notice pointing to this PR in  UPDATING.md

**Comment by dpgaspar** [comment]
Good points, I'll add both. I think we don't have a doc describing what roles are, I'll create one on the docs if it does not exist

**Review by villebro — APPROVED** [review]
I agree with the scope of Alpha permissions. This also needs a rebase.

**Review by mistercrunch — APPROVED** [review]

**Pull request merged**